burotel/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "indico-plugin-burotel",
+  "main": "indico_burotel/client/index.js",
```

Comment:
I don't think this is particularly useful since we don't use it as a node package.

I'd add `"private": true,` though to make it clear it's not a package meant to be published.